middleware.ts

```diff
@@ -17,6 +17,14 @@ export async function middleware(req: NextRequest) {
     return res;
   }
 
+  if (req.nextUrl.pathname.startsWith("/api")) {
+    return res;
+  }
+
+  if (req.nextUrl.pathname.startsWith("/assets")) {
+    return res;
+  }
```

Comment:
Refactor (non-blocking):

```suggestion
  if (/^\/(api|assets)/.test(req.nextUrl.pathname)) {
    return res;
  }
```